Samples/webapp-python/program.py

```diff
@@ -0,0 +1,287 @@
+from operator import setitem
+from re import S
+from tkinter import NO
+import requests
+import os
+import json
+import base64
+import struct
+from uuid import UUID
+from typing import Optional
+
+from objects import (
+    AggregatedValues,
+    HeightAndWeightMeasurement,
+    PublicClient,
+    PublicDataSample,
+    PublicExerciseType,
+    Resistance,
+    PublicMotion,
+    PublicMotionGroup,
+    PublicSet,
+    PublicSetData,
+    PublicExercise,
+    PublicSession,
+    SessionInfo
+)
+
+class ApiClient:

```

Comment:
Can you move this class to a separate file? I think it would improve readability a bit